### ISSUE TITLE
Seems like a "/" is included

### ISSUE THREAD

**Issue by MidyGamy** [vargaviktor/3dsalt#1]
So I installed the .unistore file and tried to select it, but I have this error:
![2024-12-01_14-49-50 736_top](https://github.com/user-attachments/assets/4c0164d7-efcb-4607-a305-11c998bbc4c9)

**Comment by vargaviktor** [comment]
Do you get this error with the latest version of Universal Updater?
I couldn't reproduce the problem.

**Comment by MidyGamy** [comment]
let me try

**Comment by MidyGamy** [comment]
Yup, still not working, and trying to get it through the QR code just crashes the console

**Comment by vargaviktor** [comment]
I am checking the error message in the source code of Universal Updater, and will correct the file.
I also found the QR code crash bug, but it is not content specific.

**Comment by vargaviktor** [comment]
From the source code UU, it seem sto me, that this shall be an / in the Storeinfo tag, but maybe I undarsand it wrong.
However, I removed all the slashes from all the files tags

Can you try it now? (revision 148)
(Maybe it is completely wrong.)

**Comment by MidyGamy** [comment]
> From the source code UU, it seem sto me, that this shall be an / in the Storeinfo tag, but maybe I undarsand it wrong.
> However, I removed all the slashes from all the files tags
> 
> Can you try it now? (revision 148)
> (Maybe it is completely wrong.)

Let me try

**Comment by vargaviktor** [comment]
Would you like to try these too?
GB(C) Homebrew DB - https://github.com/vargaviktor/database-unistore
GBA Homebrew DB - https://github.com/vargaviktor/games-unistore
(The icons are not fine yet, because my other computer with Windows dev tools in the service, I will correct the icons on next week.)

**Comment by MidyGamy** [comment]
Everything is working!
(Well except the icons of course, your ready to reapply it)

**Comment by MidyGamy** [comment]
Although cia install seems to be broken, 3dsx works perfectly

**Comment by vargaviktor** [comment]
I will correct them soon. Just wanted to find out, what is the problem.

**Comment by vargaviktor** [comment]
I think I cleaned all up finaly, and added a lot of new things.
Can you check it now, that your cia is fine or not? (Stil dont know what you are looking for. :) )

**Comment by MidyGamy** [comment]
Well it's working now (well exept those where its says it's broken ofc...)

**Comment by vargaviktor** [comment]
The GB and GBA now with icons is completely fine.
I found some CIAs not installing as you sai, I will test all the links. Also I'm on adding icons.

**Comment by vargaviktor** [comment]
Hi MidyGamy.
I tried to correct most of the CIA too.
Also a lot of screenshots and a few new entries were added. I am closing this issue now.
